public/locales/en/common.json

```diff
@@ -1,6 +1,57 @@
 {
-  "home": "Home",
   "contact": "Contact",
   "copyright": "Copyright",
-  "support": "Support"
+  "support": "Support",
+  "es": "Spanish",
+  "en": "English",
+  "routes": {
+    "/": "Home",
+    "voters": "Voters",
```

Comment:
We should call Voters Members or Eden Members , vote and voters seems a beet confusing to me #64